utopia-shared/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "utopia-shared",
+  "version": "1.0.0",
+  "description": "A shared module between the editor and Remix",
+  "scripts": {
+    "build": "tsc"
```

Comment:
This is not currently used since the code is being imported as-is (no need to build it and use it as a package)